index.js

```diff
@@ -95,7 +95,7 @@ function fastifyCors (fastify, opts, next) {
   function configureOrigin (req, reply, callback) {
     var reqOrigin = req.headers.origin
     if (isOriginFunction) {
-      var result = origin(reqOrigin, _onOrigin)
+      var result = origin(reqOrigin, fastify, _onOrigin)
```

Comment:
Why not bind the Fastify instance? Same result, no breaking change (and is also in line with what we do for hooks, middlewares...)

Reply:
See https://github.com/fastify/fastify-cors/issues/5#issuecomment-429162101

Reply:
I'm +1 for binding.